packages/code/src/Code.tsx

```diff
@@ -169,16 +280,55 @@ function Code({
     </Syntax>
   );
 
+  function handleScroll(e: React.UIEvent) {
+    const scrollWidth = e.currentTarget.scrollWidth;
+    const elementWidth = e.currentTarget.clientWidth;
+    const isScrollable = scrollWidth > elementWidth;
+
+    if (isScrollable) {
+      const scrollPosition = e.currentTarget.scrollLeft;
+      const maxPosition = scrollWidth - elementWidth;
+      setCanScrollLeft(scrollPosition > 0);
+      setCanScrollRight(scrollPosition < maxPosition);
+    }
+  }
+
   if (!multiline) {
     return (
       <div className={wrapperStyle}>
         {renderedWindowChrome}
 
         <div
-          {...(rest as DetailedElementProps<HTMLDivElement>)}
-          className={wrapperClassName}
+          className={css`
```

Comment:
Not sure if this needs to be in a separate variable since it's so short (and because I couldn't think of a good variable name), let me know what y'all think

Reply:
Should be fine to leave as-is when it's just one rule 👍